demo/src/plot_demo.rs

```diff
@@ -223,6 +230,11 @@ impl LineDemo {
                         }
                     });
             });
+
+            ui.vertical(|ui| {
+                ui.checkbox(gradient, "Gradient line");
+                ui.checkbox(gradient_fill, "Gradient fill");
```

Comment:
gradient fill only works if gradient line is also checked, which is very confusing

Reply:
Addressed by disabling the fill checkbox unless the gradient line is selected